R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1173,7 +1173,7 @@ test_that("group by, agg functions", {
 
   expect_equal(3, count(mean(gd)))
   expect_equal(3, count(max(gd)))
-  expect_equal(30, collect(max(gd))[1, 2])
+  expect_equal(30, collect(max(gd))[2, 3])
```

Comment:
I am not sure I understand why this line changed in this PR ? To debug this you can print the output with something like `debug <- collect(max(gd)); print(debug)`